lang/en_US.json

```diff
@@ -929,5 +927,33 @@
     "Similar Games": "Similar Games",
     "Progression": "Progression",
     "Win Condition": "Win Condition",
-    "Missable": "Missable"
+    "Missable": "Missable",
+    "metaDeveloper_other": "Developers",
+    "metaDeveloper_one": "Developer",
+    "metaPublisher_other": "Publishers",
+    "metaPublisher_one": "Publisher",
+    "metaGenre_other": "Genres",
+    "metaGenre_one": "Genre",
+    "metaLanguage_other": "Languages",
+    "metaLanguage_one": "Language",
+    "metaTheme_other": "Themes",
+    "metaTheme_one": "Theme",
+    "metaPerspective_other": "Perspectives",
+    "metaPerspective_one": "Perspective",
+    "metaFeature_other": "Features",
+    "metaFeature_one": "Feature",
+    "metaCredit_other": "Credits",
+    "metaCredit_one": "Credit",
+    "metaTechnical": "Technical",
+    "metaMisc": "Misc.",
+    "metaMeta": "Meta",
+    "metaProtagonist_other": "Protagonists",
+    "metaProtagonist_one": "Protagonist",
+    "metaSetting_other": "Settings",
+    "metaSetting_one": "Setting",
+    "metaRegional": "Regional",
+    "metaHackOf": "Hack of",
+    "metaEvents": "Events",
+    "metaFormat": "Format",
+    "metaRaFeature": "Achievement Extras"
```

Comment:
http://localhost:64000/game2/27576
`Set with DLC support` could be an Achievement Extra

Reply:
Agreed. Renaming the hub from "[Feature -" to "[Meta -" will cause this to automatically happen. I believe this rename is appropriate based on the current hub naming conventions of meta hubs vs feature hubs.

I've made this change on prod as well: https://retroachievements.org/manage/hubs/3484